### ISSUE TITLE
Proxy problem

### ISSUE THREAD

**Issue by fgisslen** [InstaPy/InstaPy#2456]
Hi, 
when using a proxy I get this error:

InstaPy/instapy/instapy.py", line 276, in set_selenium_local_session
    proxy = ":".join([self.proxy_address, self.proxy_port])
TypeError: sequence item 1: expected str instance, int found

**Comment by kairosooo** [comment]
same problem.

Donwloaded brand new instapy.master, only inputted username password and change the quickstart.py adding the proxy thing by following.

(Line 19)    session = InstaPy(username=insta_username,
                  password=insta_password,
                  proxy_address='196.xxx.xxx', proxy_port=600xx)

then the console shows TypeError: sequence item 1: expected str instance, int found

the current readme and instruction about proxy is poor.

**Comment by Areahints** [comment]
hello @kairosooo where you able to configure your proxy correctly? im about to try one and need guidance

**Comment by fgisslen** [comment]
I found the problem for this one. You have to cast str(self.proxy_port)

**Comment by Areahints** [comment]
how do i do that? i add it to which file? eli5 please?

**Comment by fgisslen** [comment]
@Areahints InstaPy/instapy/instapy.py", line 276 . 
proxy = ":".join([self.proxy_address, self.proxy_port])
should instead be proxy = ":".join([self.proxy_address, str(self.proxy_port)])

**Comment by Areahints** [comment]
@uluqulu @charlesccc might this be something needed to patch into master?

**Comment by CharlesCCC** [comment]
@fgisslen @Areahints Can any one of you PR the change please, so we can review it and then merge/path into master. thank you.
